packages/picker/src/MobileController.ts

```diff
@@ -26,6 +27,21 @@ export class MobileController extends InteractionController {
 
     public override handlePointerdown(): void {
         this.preventNextToggle = this.open ? 'yes' : 'no';
+        if (isWebKit()) {
+            this.target.classList.add('remove-focus-ring-safari-hack');
```

Comment:
nit: we could put this class name in a `const`

Reply:
do i keep this const here in this file only? also any naming suggestions?

Reply:
done!